opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -501,7 +521,7 @@
             "properties": {
                 "mode": "Incremental",
                 "templateLink": {
-                    "uri": "[concat(parameters('_artifactsLocation'), '/daemonvm/azuredeploy.json', parameters('_artifactsLocationSasToken'))]",
+                    "uri": "[concat(parameters('_artifactsLocation'), '/daemonvm/azuredeploy.jsonc', parameters('_artifactsLocationSasToken'))]",

```

Comment:
What was the outcome of the jsonc experiment? Have we got some jsonc and some json?

Reply:
Yup I've moved the Deamon VM to `.jsonc` as was testing and all works fine. Don't know why the ARM team didn't mention that.